src/App/Components/DisplayArticle/readArticle.js

```diff
@@ -137,22 +138,30 @@ export class ReadArticle extends Component {
             <BreadCrumb category={category} />
             <div className="container article">
               <div className="row">
-                <ArticleBody
-                  title={title}
-                  body={body}
-                  readTime={readTime}
-                  tags={tags}
-                  createdAt={createdAt}
-                  coverImage={coverImage}
-                  firstName={firstName}
-                  lastName={lastName}
-                  profileImage={profileImage}
-                  username={username}
-                  authorCredential={authorCredential}
-                />
-                {username && <Bookmark username={username} slug={slug} bookmarks={bookmarks} />}
-                <Comment slug={slug} />
-                <BottomPopular articles={articles} />
+                <div className="col-lg-1 col-sm-12 large-share">
+                  <ArticleSocialSharing slug={slug} title={title} />
+                  {username && <Bookmark username={username} slug={slug} bookmarks={bookmarks} />}
+                </div>
+                <div className="col-lg-11 col-sm-12">
+                  <ArticleBody
+                    title={title}
+                    body={body}
+                    readTime={readTime}
+                    tags={tags}
+                    createdAt={createdAt}
+                    coverImage={coverImage}
+                    firstName={firstName}
+                    lastName={lastName}
+                    profileImage={profileImage}
+                    username={username}
+                    authorCredential={authorCredential}
+                  />
+                  <div className="small-share">
+                    <ArticleSocialSharing slug={slug} title={title} />
+                  </div>
+                  <Comment slug={slug} />
+                  <BottomPopular articles={articles} />
```

Comment:
@murediane should this be here? I saw your team-mates removing the `BottomPopular articles={articles} />` line in their PRs

Reply:
It  was because the popular part was still work in progress but now that  is  done .we have to have it there